tests/unit_tests.py

```diff
@@ -123,7 +123,23 @@ def test_write_pdf(self):
     # Convert Markdown to PDF using md2pdf
         md2pdf(pdf_file_path, md_content)
 
-    def test_write_3dthesis(self):
+    def test_write_adamantine(self):
+        path_to_example_data = os.path.join(os.path.dirname(__file__), '../examples/SS316L.json')
+        mat = mist.core.MaterialInformation(path_to_example_data)
+        current_dir = os.path.dirname(os.path.abspath(__file__))
+        file = "mistinput.info"
+        adamantine_file_path = os.path.join(current_dir, 'mistinput.info')
+        try:
+            # Call the function that should create the file
+            mat.write_adamantine_input(adamantine_file_path)
+            
+            # Assert that the file now exists
+            self.assertTrue(os.path.exists(adamantine_file_path), f"File {file} should exist.")
+        
+        except AssertionError as e:
+            self.fail(f"AssertionError occurred: {str(e)}")
+
+def test_write_3dthesis(self):
```

Comment:
This test is ignored because of the spacing (I would have expected an error though)

Reply:
Okay I'll fix that.  Thanks!